.github/workflows/tests.yml

```diff
@@ -0,0 +1,68 @@
+name: "Tests"
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  test:
+    name: PHP ${{ matrix.php }} php-invoker:${{ matrix.php_invoker }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php:
+          - "5.3"
+          - "5.4"
+          - "5.5"
+          - "5.6"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
+          - "8.0"
+          - "8.1"
```

Comment:
"8.1" is duplicate here, already set up via "include below (line 29 currently):
- https://github.com/zf1s/phpunit/blob/d7ac301264cae539206442f58df1d21c4dcd5cc3/.github/workflows/tests.yml#L29-L35

Reply:
thanks @glensc , the newest version should be fine :)